www/src/components/graphiql-embed.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useRef, useCallback, useLayoutEffect } from "react"
+
+const VIDEO_RATIO = 2 / 3
+
+const GraphiQLEmbed = ({ title, url, query }) => {
+  const encodedQuery = encodeURIComponent(query)
+
+  const [iframeWidth, setIframeWidth] = useState(0)
+  const iframeRef = useRef()
+
+  const handleResize = useCallback(
+    () => setIframeWidth(iframeRef.current.clientWidth),
+    [iframeRef.current]
+  )
+
+  useLayoutEffect(() => {
+    handleResize()
+    window.addEventListener(`resize`, handleResize)
+
+    return () => {
+      window.removeEventListener(`resize`, handleResize)
+    }
+  }, [iframeRef.current])
+
+  return (
+    <iframe
+      ref={iframeRef}
+      className="graphiql-embed"
+      title={title}
+      src={`${url}?query=${encodedQuery}&explorerIsOpen=false`}
+      width={iframeWidth}
+      height={iframeWidth * VIDEO_RATIO}
```

Comment:
I know I said to copy the code from `EggheadEmbed` but this needs some adjustments to make it work. For one, the GraphQL embeds have a fixed width and height originally, but now they're blown up to the size of the screen. We'd need to ask someone to see if they're okay with the change in behavior (I don't know who to ask though).

Also, `VIDEO_RATIO` doesn't make sense as a variable name here because the embed isn't a video :P